os/HowToUseDutils.md

```diff
@@ -0,0 +1,208 @@
+
+The user can call dutils.sh with "tool" command, followed by the actual toolchain command to be run.
+```
+./dutils.sh tool nm build/output/bin/tinyara.axf
+./dutils.sh tool addr2line -e build/output/bin/tinyara.axf 0x0123abcd
+```
+
+The toolchain commands can also be run from the dutils as below:
+
+```
+1. Go to os/ folder of the TizenRT repository
+2. Run ./dutils.sh
+3. Press 2
+4. Select the desired option
+```
+
+
+### Example
+```
+test@Focal:~/Desktop/product_Refrigerator/.tizenrt/os$ sudo ./dutils.sh 
+Check Docker Image
+Docker Image Version : tizenrt/tizenrt:1.5.6
+======================================================
+  "Select Option"
+======================================================
+  "1. Run TRAP Debug tool"
+  "2. Run Toolchain Command"
+  "h. Help"
+  "x. Exit"
+======================================================
+2
+make: Entering directory '/home/test/Desktop/product_Refrigerator/.tizenrt/os/tools'
+chmod 755 ../makeinfo.sh
+make: Leaving directory '/home/test/Desktop/product_Refrigerator/.tizenrt/os/tools'
+======================================================
+  "Select Binary for Toolchain Command"
+======================================================
+  "1. Kernel"
+  "2. App1"
+  "3. App2"
+  "4. Common"
+  "5. Enter manually"
+  "x. Exit"
+======================================================
+1
+======================================================
+  "Select Toolchain Command"
+======================================================
+  "1. nm"
+  "2. addr2line"
+  "3. Enter manually"
+  "x. Exit"
+======================================================
+2
+Enter address: 
+0e07844c
+Executing: arm-none-eabi-addr2line -e ../build/output/bin/tinyara.axf 0e07844c  
+net.c:?
+>> Output is stored in dutils_output_addr2line_2024.09.06-14.33.16.txt
+
+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+```
+
+## To run TRAP tool commands,
+
+TRAP tool is used to obtain complete information about a crash or assert in TizenRT code.
+For more information about TRAP tool, please refer to
+[HowToUseTrap.md](../tools/trap/HowToUseTrap.md)
+
+```
+1. copy crash logs, if any, anywhere in the TizenRT repository(Ex: os/crashlogs)
+2. Go to os/ folder of the TizenRT repository
+3. Run ./dutils.sh
+4. Press 1
+5. Select the desired option and enter the desired path
+```
+```
+NOTE:
+After selecting TRAP debug option, we get 3 options to select. 
+Option 1-> When we get crash after building tizenrt locally and trying to debug it, we just need crashlogs to produce the desired result.
+Option 2-> when other tizenrt contributer gets crash and seek your help for debugging the crash. HERE, we need the .config, output/bin/ and crashlogs  
```

Comment:
Change as below:
Option 2-> If you need to debug a binary which is not built locally in your Tizenrt codebase, then use this option to provide a different path for the binary and config files.